app/demo-stm32h7-nucleo/app-h743.toml

```diff
@@ -2,6 +2,7 @@ name = "demo-stm32h743-nucleo"
 target = "thumbv7em-none-eabihf"
 board = "nucleo-h743zi2"
 chip = "../../chips/stm32h7"
+memory = "memory-large.toml"
```

Comment:
Is this accurate for the h743? I can't remember what the difference actually was...

Reply:
Good question!

According to the H743 datasheet (DS12110 Rev 10), AXI-SRAM is only 384 Kbytes, versus the expected 512K.

However, the reference manual for the whole product line (RM0433) lists AXI SRAM as 512 Kbytes (0x24000000 - 0x2407FFFF).

I'm not sure who to believe!

Reply:
Ah, the value of 384 KBytes in DS12110 is actually for H742 chips, which we aren't using, so I think we're fine here.